content/en/docs/concepts/services-networking/service.md

```diff
@@ -184,6 +184,33 @@ An ExternalName Service is a special case of Service that does not have
 selectors and uses DNS names instead. For more information, see the
 [ExternalName](#externalname) section later in this document.
 
+### Endpoint Slices (Alpha)
```

Comment:
This title will put the feature state into the URI for this fragment.

Instead, use a [feature state shortcode](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#feature-state) or explain the status in the main text.

Reply:
Done, thanks!